Automattic/vip-go-mu-plugins
# Adding features to validate-attachments CSV

New Features:

* Ability to skip the edge cache by appending a random query parameter to the URL
* Ability to log empty files, listed as 0 bytes or below (`-1` sometimes)
* Outputs extra information about the command running and the flags used, for potential logging

Bugs Fixed:

* Running with `--extra-check` may not produce a correct URL for multisite sites that are on a subdirectory.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Example:

1. Check out PR.
1. Run `wp vip-migrate validate-attachments` with the arguments of your choice.
1. Verify output STDOUT and CSV are as expected.
